document autocapture.networkTracking for Browser SDK (#680)
amplitude/amplitude-docs: content/collections/browser_sdk/en/browser-sdk-2.md

    networkTracking: true, //[tl! highlight]
  },
});
```

#### Advanced configuration for network tracking

Set `config.autocapture.networkTracking` to a `NetworkTrackingOptions` to configure which network requests get tracked.

{{partial:collapse name="NetworkTrackingOptions"}}

| Name                                                          | Description                          | Value                                                                                                                                                                                                                                                                            |
| ------------------------------------------------------------- | ------------------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| `captureRules` | The rules for capturing network requests. You should always append rules with specific hosts to the bottom of the list. | `undefined` |

> | `ignoreHosts` | The hosts to ignore. Supports wildcard characters `*`. eg. `["*"]` to ignore all hosts, `["*.notmyapi.com", "notmyapi.com"]` to ignore `notmyapi.com` and all subdomains. | `[]` |

Reviewer: ```suggestion
| `ignoreHosts` | The hosts to ignore. Supports wildcard characters `*`. For example, `["*"]` to ignore all hosts, `["*.notmyapi.com", "notmyapi.com"]` to ignore `notmyapi.com` and all subdomains. | `[]` |
```

| `ignoreAmplitudeRequests` | Whether to ignore Amplitude requests. | `true` |

{{/partial:collapse}}

{{partial:collapse name="NetworkTrackingOptions.NetworkCaptureRule"}}

| Name |  Description | Default Value |
| --- | --- | --- |

> | `hosts` | The hosts to capture. Supports wildcard characters `*`. eg. `["*"]` to match all hosts, `["*.example.com", "example.com"]` to match `example.com` and all subdomains. | `none` |

Reviewer: ```suggestion
| `hosts` | The hosts to capture. Supports wildcard characters `*`. For example, `["*"]` to match all hosts, `["*.example.com", "example.com"]` to match `example.com` and all subdomains. | `none` |
```

> | `statusCodeRange` | The status code range to capture. Supports comma-separated ranges or single status codes. eg. `"0,200-299,413,500-599"` | `"500-599"` |

Reviewer: ```suggestion
| `statusCodeRange` | The status code range to capture. Supports comma-separated ranges or single status codes. For example, `"0,200-299,413,500-599"` | `"500-599"` |
```
